lib/fluent/plugin/out_split.rb

```diff
@@ -9,6 +9,7 @@ class SplitOutput < Output
     config_param :key_name, :string
     config_param :out_key, :string, :default => nil
     config_param :reserve_msg, :bool, :default => nil
+    config_param :prefix, :string, :default => nil
```

Comment:
IMHO, `extracted_key_prefix` is too complicated.
But yes, `prefix` can be mingled with "tag prefix".
What about `keys_prefix` ?

Choice is yours :P